app/scripts/AddTrackModal.js

```diff
@@ -148,23 +156,25 @@ class AddTrackModal extends React.Component {
         show={this.props.show}
       >
         <Modal.Header closeButton>
-          <Modal.Title>{'Add Track'}</Modal.Title>
+          <Modal.Title>Add Track</Modal.Title>
         </Modal.Header>
         <Modal.Body>
           { form }
           {
-            this.props.hidePlotTypeChooser ? null :
-            <PlotTypeChooser
-              ref={(c) => { this.plotTypeChooser = c; }}
-              datatypes={this.state.selectedTilesets.map(x => x.datatype)}
-              onPlotTypeSelected={this.handlePlotTypeSelected.bind(this)}
-              orientation={orientation}
-            />
+            this.props.hidePlotTypeChooser ? null
```

Comment:
Not critical but this can be shortened to

```javasacript
!this.props.hidePlotTypeChooser && (...)
```

Reply:
Not in this case. Notice that it's `a ? null : b`, which isn't the same as `a && b`.